libmisc/loginprompt.c

```diff
@@ -112,34 +112,6 @@ void login_prompt (const char *prompt, char *name, int namesize)
 
 	name[i] = '\0';
 
-	/*
```

Comment:
So long as you're doing this (which sounds good to me), two of the blocks above this (NOT including name[i] = '\0') can be dropped as well, right?
 
```
105         while (isgraph (*cp)) {
106                 cp++;
107         }
108
109         if ('\0' != *cp) {
110                 cp++;
111         }
```

Reply:
Thank you for pointing it out. I have removed these lines as well. Now shadow login is compatible with dosbox login.